tests/test_area.py

```diff
@@ -15,26 +15,27 @@
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
-from pendulum import duration, today
+import unittest
 from collections import OrderedDict
 from unittest.mock import MagicMock
-import unittest
+
+from d3a_interface.constants_limits import ConstSettings, GlobalConfig
 from parameterized import parameterized
+from pendulum import duration, today
+
+from d3a import constants
+from d3a.constants import TIME_ZONE
```

Comment:
Based on the conversation we had, these two imports should be unified into one style, right?

Reply:
Indeed, will fix that, thanks!